models/user.go

```diff
@@ -712,19 +712,42 @@ func (u *User) GetOwnedOrganizations() (err error) {
 
 // GetOrganizations returns paginated organizations that user belongs to.
 func (u *User) GetOrganizations(opts *SearchOrganizationsOptions) error {
-	ous, err := GetOrgUsersByUserID(u.ID, opts)
-	if err != nil {
+	sess := x.NewSession()
```

Comment:
Since there is no insert/update/delete operations here, a transation is unnecessary. If you still want that, you should add `defer sess.Close()`.

Reply:
```suggestion
	sess := x.NewSession()
	defer sess.Close()
```